docs/en/serving.md

```diff
@@ -60,7 +60,7 @@ python3 -m fastchat.model.apply_delta \
   --target-model-path /path/to/vicuna-7b \
   --delta-path lmsys/vicuna-7b-delta-v1.1
 
-python3 lmdeploy/serve/turbomind/deploy.py vicuna-7B /path/to/vicuna-7b hf
+python3 -m lmdeploy/serve/turbomind/deploy.py vicuna-7B /path/to/vicuna-7b hf
```

Comment:
may change 'lmdeploy/serve/turbomind/deploy.py' to 'lmdeploy.serve.turbomind.deploy'